### ISSUE TITLE
[master] profile contains wrong settings

### ISSUE THREAD

**Issue by smartavionics** [Ultimaker/Cura#6407]
**Application version**

master as of 24/09/2019

**Platform**

Linux

**Printer**

Kosell XL

**Reproduction steps**

Updated cura/uranium to latest master branch and opened cura.

**Actual results**

Some profile settings are wrong.

**Expected results**

Profile settings should have the same values as they had before the update.

**Additional information**

Look at this screen shot and see how the values displayed in the profiles dialog do not correspond to the values shown in the settings area on the right. Line width, infill line width, infill pattern, connect infill lines, etc. are all wrong.

![Screenshot_2019-09-24_09-34-58](https://user-images.githubusercontent.com/585618/65495441-ad056b00-deae-11e9-929c-7d173a3d62e6.png)

**Comment by Ghostkeeper** [comment]
We discovered this regression on the master branch as well. Essentially if you save a new profile it's not going to remember the settings in that profile, and if you update the profile it becomes corrupt completely.

**Comment by smartavionics** [comment]
Thanks for the info. I have reverted back to recent usable master branches and will wait until this is fixed before merging from upstream again.

**Comment by smartavionics** [comment]
I'm sure that you are aware that this regression still exists in the master branch(es). Do you intend to fix it?

**Comment by fieldOfView** [comment]
I think they are planning to make the issue go away by replacing the code in the "intents" work that is going on in different branches.

**Comment by smartavionics** [comment]
Thanks, I suspected that was the plan. Until either the regression gets fixed or the intents stuff is merged and usable I can't merge any upstream changes. No big deal.

**Comment by Ghostkeeper** [comment]
Yeah the plan this week is to make the right hand side of that screen display the settings of the current profile with intents. The expected behaviour is (afaik):

* If the selected profile is an intent profile or a quality profile with default intent, show a combined view of both the intent profile and the quality profile.
* If the selected profile is a custom (quality changes) profile, show only the custom profile settings.
